playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -229,3 +229,56 @@
   tasks:
   - include: docker/upgrade.yml
     when: l_docker_upgrade is defined and l_docker_upgrade | bool and not openshift.common.is_atomic | bool
+
+- name: Drain and upgrade master nodes
+  hosts: oo_masters_to_config:&oo_nodes_to_upgrade
```

Comment:
This may not be necessary, masters are implicitly in the nodes group today, I don't think there's any way around that, so it might be best to keep this just at the masters group. IMO the serial and label vars should probably *not* be used during master upgrade, control plane should remain serial 1 host at a time.